tests/test_board.py

```diff
@@ -0,0 +1,59 @@
+from turtle import done
+from todo_app.view_model import ViewModel
+from todo_app.item import Item
+
+todo_list_name = 'To Do'
+doing_list_name = 'Doing'
+done_list_name = 'Done'
+
+todo_card_name = 'This is a card that is still to do'
+doing_card_name = 'This is a card that is in progress'
+done_card_name = 'This is a card that was completed today'
+done_yesterday_card_name = 'This is a card that was completed yesterday'
+
+def test_todo_cards_returned_from_board():
+
+    # Given
+    board = mock_board()
```

Comment:
You could look into pytest fixtures https://docs.pytest.org/en/6.2.x/fixture.html (have a look at the examples before you get put off by the wall of text) to make these tests cleaner and easier to read